tournament.js

```diff
@@ -23,17 +30,24 @@ $(document).ready(function() {
 	$('#topCard').find('.mdl-card__title').css("background","url('"+totalList[0].image_url+"') center");
 	$('#bottomCard').find('.mdl-card__title').css("background","url('"+totalList[1].image_url+"') center");
 
+	//document.querySelector('#p1').MaterialProgress.setProgress(0);
 	//$('#topCard.mdl-card__title').css("background-size",50+"px " + 70 + "%");
 	//$('#topCard.mdl-card__title').css("background-repeat","no-repeat");
 })
 
 $("#topButton").click(topButtonClick);
 $("#bottomButton").click(bottomButtonClick);
 
+function doNothing() {
```

Comment:
If this actually does nothing, can you remove it?

Reply:
yes